public/index.html

```diff
@@ -59,15 +58,11 @@
                     scale="6 6 6"></a-entity>
             </script>
         </a-assets>
+        <a-entity id="player-rig" networked character-controller="pivot: #head">
+            <a-sphere scale="0.1 0.1 0.1"></a-sphere>
+            <a-entity id="head" camera="userHeight: 1.6" personal-space-bubble networked="template:#head-template;showLocalTemplate:false;"<script src="<a-entity id="nametag" networked="template:#nametag-template;showLocalTemplate:false;"></a-entity>
```

Comment:
What's with this `"<script src="<a-entity id="nametag"`? Looks like a bad merge.